### DISCUSSION TITLE
V17.0.0 keycloak or keycloak-X ?

### DISCUSSION THREAD

**Discussion by bjolivot** [keycloak/keycloak#10256] [Q&A]
Hello!
Since v17.0.0 is using Quarkus do I need to use keycloak or keycloak.x image ?
I'm upgrading from Keycloak.x 16.1.1 image on kubernetes env.

Thank you!

**Comment by pedroigor** [comment]
The official image `keycloak` is now the new distribution. The Wildfly one is `keycloak-legacy`.

We are updating the guide too https://www.keycloak.org/server/containers.
